README.md

```diff
@@ -1,5 +1,6 @@
 # Procore JS SDK
 
+TODO: Is this circleci link available to external users?
```

Comment:
Yes, it's available. 

Just open it in incognito mode:
`<p><a href="https://circleci.com/gh/procore/js-sdk" rel="nofollow"><img src="https://camo.githubusercontent.com/12d7a80233ca71105d45428bd5f64b9c12ec44e93f1a08109a2dfc261cbc197a/68747470733a2f2f636972636c6563692e636f6d2f67682f70726f636f72652f6a732d73646b2e7376673f7374796c653d73766726636972636c652d746f6b656e3d62323466343734386261356431343831373038386430326130653134643337366531343631633630" alt="CircleCI" data-canonical-src="https://circleci.com/gh/procore/js-sdk.svg?style=svg&amp;circle-token=b24f4748ba5d14817088d02a0e14d376e1461c60" style="max-width:100%;"></a></p>`